### ISSUE TITLE
--top 10 returns completely different things than --top 5

### ISSUE THREAD

**Issue by smw1218** [sqs/used#2]
run on github.com/fitstar/falcore

$ gometalinter.v1 --disable=gocyclo --disable=vetshadow --disable=goconst --json ./... | used -top 5 ./...

gometalinter.v1 --disable=gocyclo --disable=vetshadow --disable=goconst --json ./... | used -top 10 ./...

not sure if this is expected behavior, but my expectation of "top" would be that the top of the list would be the same in subsequent runs.

**Comment by sqs** [comment]
Yes, this should be fixed. The problem is that gometalinter returns the results in a different order each time. We should sort those to get a canonical ordering before we perform usage sorting and filtering. 

(Note that this is only a sort stability issue. The results are still accurate.)

Thanks for the report!
